app/history/history.html

```diff
@@ -63,23 +65,27 @@ <h2>Comments</h2>
       </div>
     </md-input-container>
 
-    <md-input-container>
-      <label>Ratings (0-5)</label>
-      <md-select ng-model="rating" required="">
-        <md-option value="5">5</md-option>
-        <md-option value="4">4</md-option>
-        <md-option value="3">3</md-option>
-        <md-option value="2">2</md-option>
-        <md-option value="1">1</md-option>
-        <md-option value="0">0</md-option>
-      </md-select>
-    </md-input-container>
+    <div layout="row" >
+      <md-input-container max-width="30%">
+        <label>Ratings (0-5)</label>
+        <md-select ng-model="rating" required="">
+          <md-option value="5">5</md-option>
+          <md-option value="4">4</md-option>
```

Comment:
could use `ng-repeat` but no need to change it :)

Reply:
okok